src/wrappers/nats.js

```diff
@@ -75,13 +75,16 @@ function natsSubscribeCallbackMiddleware(
         const { slsEvent: natsEvent, startTime: natsStartTime } =
         eventInterface.initializeEvent(
             'nats',
-            serverHostname,
+            callback_subject,
             isRequestCall ? 'requestMessageListener' : 'subscribeMessageListener',
             'trigger'
         );
         tracer.addEvent(natsEvent);
         // Getting message data.
         const triggerMetadata = {};
+        if (serverHostname) {
+            triggerMetadata.serverHostname = serverHostname;
```

Comment:
I think you used a different naming convention in the `epsagon-node` pr

Reply:
tnx